### PR TITLE
#Resolved 1 : Added Navbar

### PR DESCRIPTION
**Pull request by VidhiBhatt01**
## What changes have you made?
Added a navbar that is responsive in all dimensions and screen sizes. Added links of different pages in the navbar.

## Checklist
Before you create this PR, confirm all the requirements listed below by checking the checkboxes [x]:

- [x] I am making a proper pull request, not spam.
- [x] I've checked the issue list before deciding what to submit.
- [x] I have checked there aren't other open Pull Requests for the same update/change
- [x] I have tested the code before submission
- [x] I am a member of GDSC JGEC
- [x] I'm Hacktoberfest'21 contributor.

## Related Issues or Pull Requests

Issue 1

## Add relevant screenshot or video (if any)

NavBar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,17 +39,40 @@
...
 	<!-- External JS -->
 	<!-- JS Lib CDN/File 1 -->
 	<!-- JS Lib CDN/File 2 -->
 
-	<!-- Main JS -->
-	<script defer src="./js/app.js"></script>
+	<!-- Main JS 
+	<script defer src="./js/app.js"></script>-->
 </head>
+	<nav class="navbar">
```
Comment (niloysikdar) [inline index.html:51]: Put this nav inside the body @VidhiBhatt01

**Review by niloysikdar — CHANGES_REQUESTED** [review]
Need some changes @VidhiBhatt01

**Comment by VidhiBhatt01** [comment]
@niloysikdar Extremely sorry for the inconvenience.

**Review by niloysikdar — CHANGES_REQUESTED** [review]
@VidhiBhatt01, you are still missing the opening body tag.

**Pull request merged**